mrp_production_add_middle_stuff/wizard/wizard_addition.py

```diff
@@ -51,5 +52,8 @@ def add_product(self):
                                                 self.product_qty,
                                                 self.production_id)
         line = mppl_obj.create(values)
-        production_obj._make_production_consume_line(line)
+        move = production_obj._make_production_consume_line(line)
+        move_obj.browse(move).action_confirm()
+        if self.production_id.state not in 'confirmed':
+            self.production_id.action_assign()
```

Comment:
But why are you doing action_assign for the complete production? Do it on the move itself.